src/YesSql.Provider.MySql/MySqlDbProviderOptionsExtensions.cs

```diff
@@ -27,7 +35,7 @@ public static class MySqlDbProviderOptionsExtensions
             {
                 ConnectionFactory = new DbConnectionFactory<MySqlConnection>(connectionString, true),
```

Comment:
The default value is false. Remove `, true` and it will use it. Example:
https://github.com/sebastienros/yessql/blob/master/test/YesSql.Tests/SqlServerTests.cs#L17